envoy/http/persistent_quic_info.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <memory>
+
+namespace Envoy {
+namespace Http {
+
+// Store quic helpers which can be shared between connections and must live beyond the lifetime of
+// individual connections. When used in HTTP/3 upstream, it should be owned by cluster and shared
+// across its HTTP/3 connection pools. This an opaque place holder is needed so that the memory can
```

Comment:
nit: "placeholder" (English might be a worse language than C++ but it's close!)

Reply:
nit: "the memory" => "an implementation"

Reply:
done